packages/ui-toolkit/src/styleguide/wrapper.js

```diff
@@ -1,59 +1,21 @@
 import React, { Component } from 'react';
 import { ThemeProvider, injectGlobal } from 'styled-components';
+
 import theme from '../theme';
 import Base from '../base';
 import { loadedFontFamily } from '../typography';
-import 'react-codemirror2/node_modules/codemirror/mode/jsx/jsx';
+import { RootContainer } from '../layout';
+import 'codemirror/mode/jsx/jsx';
 
 const StyledBase = Base.extend`
   /* trick prettier */
   background-color: transparent;
 `;
 
-export default class Wrapper extends Component {
-  componentWillMount() {
-    // eslint-disable-next-line no-unused-expressions
-    injectGlobal`
```

Comment:
We dont need this anynore ?

Reply:
Yeah, we can just use `<RootContainer />` ;)

Reply:
Awesome !